### PR TITLE
adds fastqc 0.12.1

### PR DESCRIPTION
**Pull request by kapsakcj**
Added a dockerfile for fastqc 0.12.1. This PR will fix issue #629

Changes from previous dockerfile:

- upgraded base image to `ubuntu:focal` from `:bionic`
- added LABEL for github repo (was missing)
- updated LABELs for base.image, dockerfile.version and software.version
- lines 70-73 now download test data from StaPH-B repo instead of poojasgupta/docker-builds fork

I also made formatting changes to `fastqc/0.12.1/README.md`, cleaned up some links, and added a section for additional tools

Also deleted an unnecessary `.gitignore` file that was uploaded on accident

Pull Request (PR) checklist:
- [x] Include a description of what is in this pull request in this message.
- [x] The dockerfile successfully builds to a test target for the user creating the PR. (i.e. `docker build --tag samtools:1.15test --target test docker-builds/samtools/1.15` )
- [x] Directory structure as name of the tool in lower case with special characters removed with a subdirectory of the version number (i.e. `spades/3.12.0/Dockerfile`)
   - [x] (optional) All test files are located in same directory as the Dockerfile (i.e. `shigatyper/2.0.1/test.sh`)
- [x] Create a simple container-specific [README.md](https://github.com/StaPH-B/docker-builds/blob/master/.github/workflow-templates/readme-template.md) in the same directory as the Dockerfile (i.e. `spades/3.12.0/README.md`)
   - [x] If this README is longer than 30 lines, there is an explanation as to why more detail was needed
- [x] Dockerfile includes the recommended [LABELS](https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile#L8-L18) 
- [x] Main [README.md](https://github.com/StaPH-B/docker-builds/blob/master/README.md) has been updated to include the tool and/or version of the dockerfile(s) in this PR
- [x] [Program_Licenses.md](https://github.com/StaPH-B/docker-builds/blob/master/Program_Licenses.md) contains the tool(s) used in this PR and has been updated for any missing

### CODE REVIEW

File: fastqc/0.12.1/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+RUN fasterq-dump --threads 4 -A SRR6082043 && \
+    gzip *.fastq && \
+    mv SRR6082043_1.fastq.gz test_1.fastq.gz && mv SRR6082043_2.fastq.gz test_2.fastq.gz
+
+# new base for testing
+FROM app as test
+
+#re-instantiate these variables
+ARG FASTQC_VER
```
Comment (erinyoung) [inline fastqc/0.12.1/Dockerfile:63]: I couldn't see where the FASTQC_VER argument was used in the test layer. 

Am I missing something?

**Comment by erinyoung** [comment]
Should close https://github.com/StaPH-B/docker-builds/issues/629

File: fastqc/0.12.1/Dockerfile (hunk 1 above)
Comment (kapsakcj) [reply fastqc/0.12.1/Dockerfile:63]: You're correct, it's not being used. Doesn't hurt to keep it but I can delete it from the dockerfile if you'd like

**Review by erinyoung — APPROVED** [review]
I removed two ARG lines, but it's the same dockerfile otherwise.

Once the tests pass I will
- merge this PR
- deploy to dockerhub and quay using the '0.12.1' tag
- overwrite the 'latest' tag

**Pull request merged**